spec/controllers/miq_capacity_controller_spec.rb

```diff
@@ -96,4 +95,18 @@
       controller.bottlenecks
     end
   end
+
+  context "#reload" do
```

Comment:
my inner @chrisarcand is crying ;) please use `describe` when describing methods